Program.cs

```diff
@@ -1,15 +1,22 @@
-using System.IO;
-﻿using System;
-using System.Net.Sockets;
+using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
 using UdtSharp;
 
 namespace p2pcopy
 {
-    class Program
+    internal class Program

```

Comment:
this line is not changing behaviour, so let's not add private keyword here, we can do that in a refactoring PR

Reply:
I will fix this via new commit